src/fec_rs_fnt.h

```diff
@@ -79,16 +78,16 @@ class RsFnt : public FecCode<T> {
         // with this encoder we cannot exactly satisfy users request, we need to
         // pad n = minimal divisor of (q-1) that is at least (n_parities +
         // n_data)
-        n = this->gf->get_code_len_high_compo(n_parities + n_data);
+        this->n = this->gf->get_code_len_high_compo(n_parities + n_data);
```

Comment:
I don't think you need the `this->` prefix (same remarks for all the occurrence).

Also, `n` could have more explicit name I think (like you have `n_data`, this could be `n_something` (or something else, just not only `n`)).